public/recentMastery.php

```diff
@@ -0,0 +1,165 @@
+<?php
+require_once __DIR__ . '/../vendor/autoload.php';
+
+RA_ReadCookieCredentials($user, $points, $truePoints, $unreadMessageCount, $permissions);
+
+$maxCount = 25;
+// First day with Game Awards
+$minDate = '2014-09-29';
+
+$errorCode = requestInputSanitized('e');
+$offset = requestInputSanitized('o', 0, 'integer');
+$offset = max($offset, 0);
+$friends = requestInputSanitized('f', 0, 'integer');
+$date = requestInputSanitized('d', date("Y-m-d"));
+
+switch ($friends) {
+    case 0: // Global
+        $lbUsers = "";
+        break;
+    case 1: // Friends
+        $lbUsers = "Friends";
+        break;
+    default:
+        $lbUsers = "";
+        break;
+}
+
+if ($friends == 1) {
+    $data = getRecentMasteryData($date, null, $user, $offset, $maxCount + 1);
+} else {
+    $data = getRecentMasteryData($date, $user, null, $offset, $maxCount + 1);
+}
```

Comment:
Line 20 suggests `friends=1` will return just the entries for the user's friends, but line 29 passes `null` for `friendsOf`. Am I reading this wrong?